src/cf-base.less

```diff
@@ -297,7 +315,6 @@ dl,
 table,
 figure {
     margin-top: 0;
-    // Spacing has been eyeballed to be 30px from the baseline.
     margin-bottom: unit(20px / @base-font-size-px, em);
```

Comment:
Oops, that's what I was missing. I think we probably should split these out into the first four and the last two, and update the last two.